### PR TITLE
Disable lambdaedge for china regions

### PR DESCRIPTION
**Pull request by dengmingtong**
Issue #, if available:

Description of changes:

Checklist:

- [x] Have you successfully deployed to an AWS account with your changes?
- [ ] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully tested with your changes locally?
- [ ] If new dependencies have been added, have they been pinned to specific versions?
- [ ] Is this change also required on the AWS Solution version?
- [ ] Have you updated openapi.yaml if you made updates to API definition (including add, delete or update parameter and request data schema)?
- [ ] If you had to run manual tests, have you considered automating those tests by adding them to [end-to-end tests](../main/end-to-end-tests/README.md)?
- [ ] If you are updating the changelog and vending out a new release, have you updated versionNumber and versionDate in [.defaults.yml](../main/config/settings/.defaults.yml)

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: main/config/settings/.defaults.yml [hunk 1]
```diff
@@ -237,3 +237,6 @@ awsPartition: aws
 # China Mainland regions: 'amazonaws.com.cn'
 # Global regions: 'amazonaws.com'
 awsSuffix: amazonaws.com
+
+# Regions that do not support lambda edge
+disableLambdaEdgeRegions: '["cn-north-1", "cn-northwest-1"]'
```
Comment (zxkane) [inline main/config/settings/.defaults.yml]: CloudFront and Lambda@Edge are global service. might use partition precise?

**Review by zxkane — CHANGES_REQUESTED** [review]

File: main/config/settings/.defaults.yml (hunk 1 above)
Comment (dengmingtong) [reply main/config/settings/.defaults.yml]: I suggest we use "disableLambdaEdgeRegions" just for Lambda@Edge, so we can support the regions which partition is "aws" but it does not need or support Lambda@Edge.
The "disableLambdaEdgeRegions" config is in .default.xml file, do not need customer to set. We can maintain this config.

Comment (zxkane) [reply main/config/settings/.defaults.yml]: > we can support the regions which partition is "aws" but it does not need or support Lambda@Edge

any example? 

Lambda@Edge functions must be deployed to `us-east-1`, same as the certificate of CloudFront. That's why it's a global service not regional service.

Comment (dengmingtong) [reply main/config/settings/.defaults.yml]: Use region to check whether edge should be skip or not

**Review by zxkane — APPROVED** [review]

**Pull request merged**